pkg/asset/internal/templates.go

```diff
@@ -913,21 +913,71 @@ spec:
     - --name=boot-etcd
     - --listen-client-urls=http://0.0.0.0:12379
     - --listen-peer-urls=http://0.0.0.0:12380
-    - --advertise-client-urls=http://$(MY_POD_IP):12379
-    - --initial-advertise-peer-urls=http://$(MY_POD_IP):12380
-    - --initial-cluster=boot-etcd=http://$(MY_POD_IP):12380
+    - --advertise-client-urls=http://{{ .BootEtcdServiceIP }}:12379
```

Comment:
Yeah. I agree that the TPR manifest could be rendered. The discussion in this is about this specific line of change:

```
-    - --advertise-client-urls=http://$(MY_POD_IP):12379
+    - --advertise-client-urls=http://{{ .BootEtcdServiceIP }}:12379
```